### DISCUSSION TITLE
Open a json file from sources

### DISCUSSION THREAD

**Discussion by uwbwsvd** [twinbasic/twinbasic#268] [Q&A] [answered]
How can i open and read this file?
![grafik](https://user-images.githubusercontent.com/41230018/124627076-8942c780-de7f-11eb-9282-5e5529a92389.png)

**Comment by WaynePhillipsEA** [comment, marked as answer]
I assume you mean programmatically? 😊

You need to put the file into a resource.    Right click on the Resources folder, select 'New Folder', enter a folder name, such as 'JSON'.  Drag the test.json file into the new folder.

You can now read the data programmatically using the LoadResData function (e.g. LoadResData("test.json", "JSON"), which returns a byte array containing the data.  

Though be aware that JSON files contain UTF8 data, so if you're wanting to load the JSON into a String variable, you should convert it properly (see https://www.di-mgt.com.au/howto-convert-vba-unicode-to-utf8.html#Utf8BytesToString).  If you already have a JSON parser, it probably already accepts byte-array data.
